### ISSUE TITLE
Feature Request:  Support sys/init API

### ISSUE THREAD

**Issue by TechnicalMercenary** [terraform-providers/terraform-provider-vault#632]
I don't see any existing support in the terraform provider vault for the `sys/init` API calls.

Before I head down this road of implementing support, are there any important details that I should be aware of ?  For instance, did I not see where this been done before, by someone else, or is this not possible with a TF provider?

**Comment by aykay76** [comment]
I would also like to see this to be able to complete an automated deployed of Vault using Terraform.

**Comment by dbond007** [comment]
I would too, but ended up creating an ansible role to do it (still working on it, but it will deploy a single node an initialise it). I am working on it to deploy vault to VMs via terraform, automatically create a cluster, setup TLS, return unseal keys and root clear or encrypted etc.
If anyone was working on this in terraform it would have been good, otherwise I will carry on with this. If you are interested its at: https://github.com/dbond007/ansible-role-hashicorp_vault

**Comment by ChrisJBurns** [comment]
@TechnicalMercenary  Nope, it is possible, someone has already done this [here](https://registry.terraform.io/providers/rickardgranberg/vaultoperator/latest/docs#schema), it's just missing the `unseal` functionality. Happy to work on this, providing this actually gets to go into the codebase.
